pkg/util/taints/taints_test.go

```diff
@@ -448,6 +583,21 @@ func TestReorganizeTaints(t *testing.T) {
 			expectedOperation: TAINTED,
 			expectedErr:       false,
 		},
+		{
+			name:      "not add an existing taint",
+			overwrite: false,
+			taintsToAdd: []v1.Taint{
+				{
+					Key:    "foo",
+					Value:  "bar",
+					Effect: v1.TaintEffectNoSchedule,
+				},
+			},
+			taintsToDelete:    []v1.Taint{},
+			expectedTaints:    node.Spec.Taints,
+			expectedOperation: UNTAINTED,
+			expectedErr:       false,
+		},
```

Comment:
I didn't find usages of ReorganizeTaints. Could you remove it instead?

Reply:
Can you remove the functions `ReorganizeTaints` and `addTaints` if they are not used?

Reply:
ok, I understand what you mean now, the `ReorganizeTaints ` function is not used. I will remove it.